Sources/StreamChatUI/ImageGalleryVC.swift

```diff
@@ -50,6 +50,10 @@ open class _ImageGalleryVC<ExtraData: ExtraDataTypes>: _ViewController, UIConfig
     public private(set) lazy var currentPhotoLabel = UILabel()
         .withoutAutoresizingMaskConstraints
     
+    public private(set) lazy var closeButton = _Button()
+    
+    public private(set) lazy var shareButton = _Button()
+    
```

Comment:
I wouldn't be afraid to make components for these buttons. I'm sure it's not the only place where we will need them.